QSBs/qsb-template.txt

```diff
@@ -5,22 +5,45 @@
 
                                 TODO
 
-User action required
----------------------
+User action
+------------
 
-Users must install the following specific packages in order to address
-the issues discussed in this bulletin:
+TODO:
+No special user action is required. Continue to update normally [1] in
```

Comment:
> No special user action is required

This IMHO is misleading since we don't have automatic updates. Users need to click something to receive updates. Maybe simply drop this sentence, and keep the next one ("Continue ...") ?

Reply:
My intention was to distinguish between "special" and "routine" user actions. Updating normally is a routine user action that takes place once every several days, even when there are no QSBs. By contrast, special actions are much rarer. An example would be the template reinstallation procedure described in [QSB-28](https://www.qubes-os.org/news/2016/12/19/qsb-28/).

If we just drop the first sentence, I think some readers will ask whether there is anything else they can or should do. What if we swap the order of the two sentences and clarify the second sentence a bit?

> Continue to update normally [1] in order to receive the security updates described in the "Patching" section below. No other user action is required in response to this QSB.

Reply:
Yes, this is better.